### PR TITLE
Fix resources menu display for turkish translation

### PR DESCRIPTION
**Pull request by veggiematts**
- Use unicode string comparison for privileges check

### CODE REVIEW

**Comment by veggiematts** [comment]
Fix for #500

**Comment by ozhansaglik** [comment]
@veggiematts Did you have a chance to examine problem?

**Comment by veggiematts** [comment]
The provided fix solved the problem on my instance (PHP 7.0.32)
Did anyone else had a chance to test this?

File: resources/admin/classes/domain/User.php [hunk 1]
```diff
@@ -70,7 +69,7 @@ public function isAdmin(){
 	public function canEdit(){
 		$privilege = new Privilege(new NamedArguments(array('primaryKey' => $this->privilegeID)));
 
-		if ((strtoupper($privilege->shortName) == 'ADD/EDIT') || (strtoupper($privilege->shortName) == 'ADMIN')){
+		if ((mb_strtoupper($privilege->shortName) == 'ADD/EDIT') || (strtoupper($privilege->shortName) == 'ADMIN')){
```
Comment (jeffnm) [inline resources/admin/classes/domain/User.php]: The second `strtoupper` didn't get changed. When it is also `mb_strtoupper` the issue is fixed on my test. 
```suggestion
		if ((mb_strtoupper($privilege->shortName) == 'ADD/EDIT') || (mb_strtoupper($privilege->shortName) == 'ADMIN')){
```

**Pull request merged**